contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -236,6 +242,126 @@ exec_tsql_stmt(PLtsql_execstate *estate, PLtsql_stmt *stmt, PLtsql_stmt *save_es
 	return rc;
 }
 
+static int
+exec_stmt_kill(PLtsql_execstate *estate, PLtsql_stmt_kill *stmt)
+{
+	PGPROC *proc;    	
+	Oid	    sysadmin_oid = get_role_oid("sysadmin", false);  // should really use BABELFISH_SYSADMIN in tds_int.h
```

Comment:
In Postgres/Babelfish Extensions we follow a specific pattern to write comments.
For mulit-line Comments: Ending with full stop
```
\*
 * This is
 * the comment.
 */
```
inline comments: If you start the comment with a capital letter then end it with a full stop 
```/* This is the comment. */```